.github/workflows/deploy-main.yml

```diff
@@ -8,6 +8,10 @@ on:
   push:
     branches: [ main ]
 
+env:
+  NETLIFY_VERSION: 12.0.11
+  NODE_VERSION: 18
```

Comment:
Is this recognized by renovate?

Reply:
not yet. but we could add a regex for that